components/nodemanager-service/pgdb/db.go

```diff
@@ -12,12 +12,21 @@ import (
 	"github.com/sirupsen/logrus"
 
 	"github.com/chef/automate/components/nodemanager-service/config"
+	project_update_lib "github.com/chef/automate/lib/authz"
 	"github.com/chef/automate/lib/db/migrator"
 	"github.com/chef/automate/lib/logger"
 )
 
+type ProjectUpdate struct {
+	jobStatus      project_update_lib.JobStatus
+	jobStatusError error
+	running        bool
+	ID             string
+}
+
 type DB struct {
 	*gorp.DbMap
+	ProjectUpdate *ProjectUpdate
 }
```

Comment:
It seems weird to me that the nodemanager's DB struct always has a ProjectUpdate in it now, but I can't really suggest a better way to do this.